subnet/deployment-generator/script/gen_config.js

```diff
@@ -12,19 +14,26 @@ var config = {
   network_id:      parseInt(process.env.NETWORK_ID    || Math.floor(Math.random() * (65536 - 1) + 1)),
   secret_string:   (process.env.SERVICES_SECRET       || crypto.randomBytes(10).toString('hex')),
   relayer_mode:    (process.env.RELAYER_MODE          || 'full'), //full or lite
+  docker_image_name: (process.env.IMAGE_NAME          || 'xinfinorg/subnet-generator:latest'),
   version: {
-    subnet:   (process.env.VERSION_SUBNET   || 'v0.1.3'),
-    bootnode: (process.env.VERSION_BOOTNODE || 'v0.1.3'),
+    subnet:   (process.env.VERSION_SUBNET   || 'v0.1.5'),
+    bootnode: (process.env.VERSION_BOOTNODE || 'v0.1.4'),
     observer: (process.env.VERSION_OBSERVER || 'latest'),
-    relayer:  (process.env.VERSION_RELAYER  || 'v0.1.3'),
-    stats:    (process.env.VERSION_STATS    || 'v0.1.3'),
-    frontend: (process.env.VERSION_FRONTEND || 'v0.1.3')
+    relayer:  (process.env.VERSION_RELAYER  || 'v0.1.4'),
+    stats:    (process.env.VERSION_STATS    || 'v0.1.4'),
+    frontend: (process.env.VERSION_FRONTEND || 'v0.1.4')
```

Comment:
I will bump it to the latest stable, everything latest doesn't guarantee to work
for our testing we can easy change to all latest in the initial config.

Reply:
After thinking some more this case will make a difference when the user redeploys. If they run 'docker pull' command (maybe accidentally) with the latest tag they will auto update their components. The consideration is if we want this behavior or static version. I think it's risky maybe bump version can have breaking changes, forking etc.

Reply:
good point. So better we remove this default version as well.
My point here is, if we keep updating our version. We need to come here to change again and again. Need to find a way to avoid this.

Reply:
I think it's ok, we are already the provider of stable versions. We have put the stable info somewhere. If not here, then have to find a way to pull that info. This can reduce 1 step of user manual task.